uint8 role
uint8 id
uint32 localTime
uint32 systemTime
float32 voltage
float32[3] pos
float32[3] eop
float32[3] vel
float32[3] angle
float32[4] q
float32[3] imuGyro
float32[3] imuAcc
LinktrackNode2[] node
